From 4ff47c214b2fe4e0536b21f25c1adb25f0081634 Mon Sep 17 00:00:00 2001
From: Jun Zhao <mypopydev@gmail.com>
Date: Wed, 21 Nov 2018 14:07:48 +0800
Subject: [PATCH 2/2] Changelog: Add new entry for svt-hevc encoder

Signed-off-by: Huang, Zhengxu <zhengxu.huang@intel.com>
Signed-off-by: hassene <hassene.tmar@intel.com>
Signed-off-by: Jun Zhao <jun.zhao@intel.com>
---
 Changelog | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Changelog b/Changelog
index 1f53ff46da..eca63270c5 100644
--- a/Changelog
+++ b/Changelog
@@ -10,7 +10,7 @@ version <next>:
 - truehd_core bitstream filter
 - dhav demuxer
 - PCM-DVD encoder
-
+- add SVT(Scalable Video Technology) HEVC encoder
 
 version 4.1:
 - deblock filter
-- 
2.17.1

